src/components/WalletSelector/WalletSelector.tsx

```diff
@@ -24,17 +26,26 @@ function WalletSelector() {
     error,
   } = context;
 
+  const headerMessage = active ? 'Log in with wallet' : 'Connect your wallet';
+  let signer;
+  if (library && account) {
+    signer = library.getSigner(account);
+  }
```

Comment:
prefer to useMemo here instead of setting a variable on the fly, like:
```js
const signer = useMemo(() => {
  return (library && account) ? library.getSigner(account) : undefined
}, [library, account])
```

Reply:
done